build.gradle.kts

```
@@ -58,20 +74,25 @@ gradlePlugin {
     }
 }
 
-fun getTestParameter(envName: String, propertyName: String, defaultValue: String? = null): String {
-    return System.getenv(envName)
+fun getTestParameter(
+    envName: String,
+    propertyName: String,
+    defaultValue: String? = null,
+): String =
+    System.getenv(envName)
         ?: (project.findProperty(propertyName) as String?)
```

detekt (1.23.8)

[warning] Use the safe cast ('as? String') instead of 'as String?'.

(detekt.potential-bugs.CastToNullableType)